spec/requests/stories_index_spec.rb

```diff
@@ -418,7 +418,7 @@ def sets_remember_token
       let(:tag) { create(:tag) }
 
       it "renders tag index properly with many posts", :aggregate_failures do
```

Comment:
I think we should move the whole `describe "GET tag index" do` section to its own file